octopi.js

```diff
@@ -35,4 +43,29 @@ Octopi.prototype = {
       return tbl[id];
     });
   }
+  /**
+     * Serialize the Octopi trie as string
+     *
+     * @return String
+     */
+  serialize: function(){
+    
+    var o = new Object();
```

Comment:
No particular reason! Feel free to make some changes :)

-- 
Celine Bursztein
------------------------------------------------------
http://celine.im
@cealtea <https://twitter.com/cealtea> on Twitter, +Celine Bursztein
<https://plus.google.com/+Celine%20Bursztein> on Google+

On Mon, Sep 5, 2016 at 6:31 PM Eeo Jun <notifications@github.com> wrote:

> In octopi.js
> <https://github.com/eugene-eeo/octopi/pull/2#discussion_r77563095>:
>
> > @@ -35,4 +43,29 @@ Octopi.prototype = {
> >        return tbl[id];
> >      });
> >    }
> > +  /**
> > +     * Serialize the Octopi trie as string
> > +     *
> > +     * @return String
> > +     */
> > +  serialize: function(){
> > +
> > +    var o = new Object();
>
> Hey, any reason why this is used instead of an object literal?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eugene-eeo/octopi/pull/2/files/04687289913bb58d39c468eada293fa7926d5efa#r77563095>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABICDhkbCJ5eZLJ1ctIhSpSilTWwmqmdks5qnMJpgaJpZM4J1TFC>
> .
>